src/app/app.component.scss

```diff
@@ -1,17 +1,50 @@
+$font: "Open Sans", sans-serif;
```

Comment:
We can move these to helper.scss ?

Reply:
Doesn't work when you put it out of the component's context.. the variables don't get shared across files.

Reply:
Oh right, you have to import it to use the variables in another file.. We should make a common scss file that we can import :)